### PR TITLE
Tidy “Perform a Rollback on a DaemonSet”

### PR DESCRIPTION
**Pull request by sftim**
Tidy the page [Perform a Rollback on a DaemonSet](https://kubernetes.io/docs/tasks/manage-daemon/rollback-daemon-set/)

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] Tidy “Perform a Rollback on a DaemonSet”

**Comment by sftim** [comment]
/retitle Tidy “Perform a Rollback on a DaemonSet”

**Comment by sftim** [comment]
Relevant to #4513

File: content/en/docs/tasks/manage-daemon/rollback-daemon-set.md [hunk 1]
```diff
@@ -123,20 +125,20 @@ resources:
...
 
-Each `ControllerRevision` stores the annotations and template of a DaemonSet
+Each ControllerRevision stores the annotations and template of a DaemonSet
 revision. The name of a ControllerRevision object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
-`kubectl rollout undo` takes a specific `ControllerRevision` and replaces
-DaemonSet template with the template stored in the `ControllerRevision`.
+`kubectl rollout undo` takes a specific ControllerRevision and replaces
+DaemonSet template with the template stored in the ControllerRevision.
```
Comment (kbhawkey) [inline content/en/docs/tasks/manage-daemon/rollback-daemon-set.md:141]: nit: the DaemonSet template?

File: content/en/docs/tasks/manage-daemon/rollback-daemon-set.md [hunk 2]
```diff
@@ -4,28 +4,29 @@ reviewers:
...
 
 
 {{% capture prerequisites %}}
 
-* The DaemonSet rollout history and DaemonSet rollback features are only
-  supported in `kubectl` in Kubernetes version 1.7 or later.
-* Make sure you know how to [perform a rolling update on a
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+You should already know how to [perform a rolling update on a
```
Comment (kbhawkey) [inline content/en/docs/tasks/manage-daemon/rollback-daemon-set.md:21]: nit: `... You should already know ...`
Possibly reword to something like: 
This task assumes that you understand how to [perform ....]().

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-21080--kubernetes-io-master-staging.netlify.app/docs/tasks/manage-daemon/rollback-daemon-set/

**Comment by kbhawkey** [comment]
Squash commits?

File: content/en/docs/tasks/manage-daemon/rollback-daemon-set.md [hunk 3]
```diff
@@ -4,29 +4,24 @@ reviewers:
...
 
-This page shows how to perform a rollback on a DaemonSet. 
-
-
+This page shows how to perform a rollback on a {{< glossary_tooltip term_id="daemonset" >}}.
 
 
 ## {{% heading "prerequisites" %}}
 
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (kbhawkey) [inline content/en/docs/tasks/manage-daemon/rollback-daemon-set.md:17]: nit: The text rendered by the include is formatted differently.

Comment (tengqm) [reply content/en/docs/tasks/manage-daemon/rollback-daemon-set.md:17]: noticed that too. may need some eyes on it.

**Pull request merged**